src/Form/AutologoutSettingsForm.php

```diff
@@ -248,21 +248,20 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
 
     if($values['role_logout']) {
       // Validate timeouts for each role.
-      foreach (user_roles(TRUE) as $key => $role) {
-        if (empty($new_stack[$key])) {
+      foreach (user_roles(TRUE) as $role => $role_object) {
+        if (empty($new_stack[$role]) || $new_stack[$role]['enabled'] == 0 ) {
```

Comment:
The other key changes are unexpected; the $role variable name looks better than $role object. The only change needed was $new_stack[$role]['enabled'] == 0
can we revert it ?